joomla/joomla-cms
# [4.1] A new Icon for com_tags to the quickicon module

Pull Request for Issue # .
Suggested by a user in the german community.

### Summary of Changes
This PR adds a new icon for com_tags to the Quickicon panel site. For users with many tags this is very useful.


### Testing Instructions
Edit the module quickicon site. 
Activate the new icon tags, if you want with counting items.

<img width="789" alt="quickicon-tags1" src="https://user-images.githubusercontent.com/1035262/106395442-922bfa80-6402-11eb-97a0-0220dfaf8555.PNG">




Then you should see the Icon in your dashboard. Make sure that ony published tags are counted 

<img width="1058" alt="quickicon-tags2" src="https://user-images.githubusercontent.com/1035262/106395463-b12a8c80-6402-11eb-85cb-d4d66708339a.PNG">


### Documentation Changes Required
yes
